x-pack/plugins/triggers_actions_ui/public/application/components/rules_setting/rules_settings_flapping_form_section.tsx

```diff
@@ -0,0 +1,213 @@
+      </div>
+    );
+  };
+
+  return (
+    <EuiFormRow label={renderLabel()}>
+      <EuiRange
+        min={min}
+        max={max}
+        step={1}
+        value={value}
+        disabled={disabled}
+        onChange={onChange}
+        showLabels
+        showValue
+        {...rest}
+      />
+    </EuiFormRow>
+  );
+});
+
+export interface RulesSettingsFlappingFormSectionProps {
+  flappingSettings: RulesSettingsFlappingProperties;
+  compressed?: boolean;
+  onChange: (key: OnChangeKey, value: number) => void;
+}
+
+export const RulesSettingsFlappingFormSection = memo(
+  (props: RulesSettingsFlappingFormSectionProps) => {
+    const { flappingSettings, compressed = false, onChange } = props;
+
+    const { lookBackWindow, statusChangeThreshold } = flappingSettings;
+
+    const {
+      application: { capabilities },
+    } = useKibana().services;
+
+    const {
+      rulesSettings: { writeFlappingSettingsUI },
+    } = capabilities;
+
+    const canWriteFlappingSettings = writeFlappingSettingsUI;
+
+    return (
+      <EuiFlexGroup direction="column">
+        {compressed && (
+          <>
+            <EuiFlexItem>
+              <EuiFlexGroup direction="column" gutterSize="s">
+                <EuiFlexItem>
+                  <RulesSettingsFlappingTitle />
+                </EuiFlexItem>
+                <EuiFlexItem>
+                  <RulesSettingsFlappingDescription />
+                </EuiFlexItem>
+                <EuiSpacer size="s" />
+              </EuiFlexGroup>
+            </EuiFlexItem>
+          </>
+        )}
+        <EuiFlexItem grow={false}>
+          <RulesSettingsRange
+            data-test-subj="lookBackWindowRangeInput"
+            min={MIN_LOOK_BACK_WINDOW}
+            max={MAX_LOOK_BACK_WINDOW}
+            value={lookBackWindow}
+            onChange={(e) => onChange('lookBackWindow', parseInt(e.currentTarget.value, 10))}
+            label={lookBackWindowLabel}
+            disabled={!canWriteFlappingSettings}
+          />
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <RulesSettingsRange
+            data-test-subj="statusChangeThresholdRangeInput"
+            min={MIN_STATUS_CHANGE_THRESHOLD}
+            max={MAX_STATUS_CHANGE_THRESHOLD}
+            value={statusChangeThreshold}
+            onChange={(e) => onChange('statusChangeThreshold', parseInt(e.currentTarget.value, 10))}
+            label={statusChangeThresholdLabel}
+            disabled={!canWriteFlappingSettings}
+          />
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiPanel borderRadius="none" color="subdued">
+            <EuiText size="s">
+              <FormattedMessage
+                id="xpack.triggersActionsUI.rulesSettings.flapping.flappingSettingsDescription"
+                defaultMessage="An alert will be considered flapping if it changes status {statusChangeThreshold} within the last {lookBackWindow}."
```

Comment:
Another option (with the inclusion of "at least"):
```suggestion
                defaultMessage="If an alert changes status at least {statusChangeThreshold} within the last {lookBackWindow}, it is flapping."
```

Reply:
Addressed in https://github.com/elastic/kibana/pull/149086